OneCommunityGlobal/HighestGoodNetworkApp
# Johny fix bug create button earned dates for badges

# Description
Fixe bug ( PRIORITY MEDIUM) - 17
Need a “Details” button or other way to see when specific badges were earned. Some badges are earned multiple times, so the button needs to show all dates when they were earned

## Mainly changes explained:
1 -  added a  dropdown button  in the modal that show all earned badges
2 - Was added some lines of code in the badgeManagment.js file, to get the specified date when the badge is earned.

…

## How to test:
check into current branch
do `npm install` and `...` to run this PR locally
log as admin user (or any user)
go to dashboard→ scroll the screen→ Badges→ Badge Reports Button.
Will open a modal and will show the new functionality.
![Captura de Tela (95)](https://user-images.githubusercontent.com/39320057/210151423-447fdd54-b2b3-46e8-a185-1c860889d72b.png)
[Video ](https://www.dropbox.com/s/8eylgss9bbe9iph/2022-12-31%2013-35-13.mp4?dl=0)

## Note:
To see this functionality you need to verify the pull request that I made for the backend too.
